test/scraper/bayern_landtag_scraper_test.rb

```diff
@@ -0,0 +1,152 @@
+    assert_equal 482, items.length
+  end
+
+  test 'extract title from item row' do
+    body = @scraper.extract_body(@detail)
+    item = @scraper.extract_detail_item(body)
+    title = @scraper.extract_title(item)
+    assert_equal 'Umzug der Schulaufsicht von Perleberg nach Neuruppin', title
+  end
+
+  test 'extract originators' do
+    body = @scraper.extract_body(@detail)
+    item = @scraper.extract_detail_item(body)
+    meta = @scraper.extract_meta(item)
+    originators = @scraper.extract_originators(meta.text, Paper::DOCTYPE_MINOR_INTERPELLATION)
+    assert_equal 'Gordon Hoffmann', originators[:people][0]
+    assert_equal 'CDU', originators[:parties][0]
+  end
+
+  test 'extract published_at' do
+    body = @scraper.extract_body(@detail)
+    item = @scraper.extract_detail_item(body)
+    meta = @scraper.extract_meta(item)
+    published_at = @scraper.extract_published_at(meta.text)
+    assert_equal Date.parse('2015-02-13'), published_at
+  end
+
+  test 'extract overview paper' do
+    body = @scraper.extract_body(@overview)
+    item = @scraper.extract_overview_items(body).last
+    paper = @scraper.extract_paper_overview(item)
+
+    assert_equal(
+      {
+        legislative_term: '6',
+      full_reference: '6/39',
+      reference: '39',
+      url: 'http://www.parldok.brandenburg.de/parladoku/w6/drs/ab_0001/39.pdf',
+      published_at: Date.parse('2014-11-03'),
+      is_answer: true
+    }, paper)
+  end
+
+  test 'extract detail paper' do
+    body = @scraper.extract_body(@detail)
+    item = @scraper.extract_detail_item(body)
+    paper = @scraper.extract_detail_paper(item)
+
+    assert_equal(
+      {
+        legislative_term: '6',
+      full_reference: '6/618',
+      reference: '618',
+      doctype: Paper::DOCTYPE_MINOR_INTERPELLATION,
+      title: 'Umzug der Schulaufsicht von Perleberg nach Neuruppin',
+      url: 'http://www.parldok.brandenburg.de/parladoku/w6/drs/ab_0600/618.pdf',
+      published_at: Date.parse('2015-02-13'),
+      originators: {
+      people: ['Gordon Hoffmann'],
+      parties: ['CDU']
+    },
+      is_answer: true
+    }, paper)
+  end
+
+  test 'extract major interpellation detail paper 614' do
+    detail = Nokogiri::HTML(File.read(Rails.root.join('test/fixtures/brandenburg_scraper_detail_614.html')).force_encoding('windows-1252'))
+    body = @scraper.extract_body(detail)
+    item = @scraper.extract_detail_item(body)
+    paper = @scraper.extract_detail_paper(item)
+
+    assert_equal(
+      {
+        legislative_term: '6',
+      full_reference: '6/614',
+      reference: '614',
+      doctype: Paper::DOCTYPE_MAJOR_INTERPELLATION,
+      title: 'Situation von Flüchtlingen und Asylbewerbern in Brandenburg',
+      url: 'http://www.parldok.brandenburg.de/parladoku/w6/drs/ab_0600/614.pdf',
+      published_at: Date.parse('2015-02-12'),
+      originators: {
+      people: [],
+      parties: ['CDU']
+    },
+      is_answer: true
+    }, paper)
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.